src/CModel.cc

```diff
@@ -727,25 +867,22 @@ class CModelAlgorithm::Impl {
         Vector amplitudes = tg.maximize();
         result.flux = expData.fitSysToMeasSys.flux * amplitudes.sum();
 
-        // To compute the uncertainty on the cmodel flux, we start by transforming the amplitude parameters
-        // by the following orthogonal matrix:
-        Matrix p(2,2);
-        p <<
-            M_SQRT1_2, -M_SQRT1_2,
-            M_SQRT1_2, M_SQRT1_2;
-        Vector p_mu = p * amplitudes;
-        Matrix p_hessian_pt = p * hessian * p.adjoint();
-        // After this transformation, \sqrt(2)*p_mu[1] is the total flux, and \sqrt(2)*p_mu[0] is the
-        // difference between the fluxes of the two components.
-        // We define the flux error as the variance on the total flux with the difference between the
-        // fluxes held fixed.  This is artificial, and an underestimate of the true uncertainty, as we
-        // ought to be marginalizing over the difference between the fluxes - but that integral would
-        // often diverge if we don't take into account the constraints, and if we do, the probability
-        // distribution we get is complicated (Gaussian times a difference of error functions) and
-        // asymmetric, so we'll leave that as a potential project for the future.
-        result.fluxSigma = expData.fitSysToMeasSys.flux / std::sqrt(p_hessian_pt(1,1));
+        // To compute the error on the flux, we treat the best-fit composite profile as a continuous
+        // aperture and compute the uncertainty on that aperture flux.
+        // That means this is an underestimate of the true uncertainty, but it's the sort that kind of
+        // makes sense for colors, and it's consistent with the fact that we're also ignoring the
```

Comment:
Can you make a quick note on why it makes sense for colors, since you are typing out a bit of a description anyway